docs/development-workflow.md

```diff
@@ -0,0 +1,72 @@
+# Development Workflow
+
+
+Start by forking the `pd` GitHub repository, make changes in a branch and then send a pull request. The development workflow consists of the following steps:
+
+1. Fork the pd repository.
+2. Make changes in a branch.
+3. Send a pull request.
+
+### Set up your pd GitHub Repository
+
+
+Fork the [pd upstream](https://github.com/pingcap/pd/fork) source repository to your personal repository.
+
+```sh
+$ mkdir -p $GOPATH/src/github.com/pingcap
```

Comment:
now we don't need to fork in the GOPATH, we can save PD to any place.

Reply:
Yes. What I expressed is to fork the pd repository to personal repository.